go/vt/servenv/http.go

```diff
@@ -53,10 +50,6 @@ func HTTPRegisterProfile() {
 		return
 	}
 
-	if !pflag.Lookup("pprof-http").Changed {
-		log.Warning("Beginning in v20, pprof-http will default to `false`; to continue enabling pprof endpoints, please manually set this flag before upgrading.")
-	}
```

Comment:
I don't think we announced it in the release notes for `v19`. But, perhaps we should add a short line to announce this in the v20 release notes